docassemble/ALToolbox/data/questions/al_income.yml

```
@@ -89,4 +89,602 @@ data: !!omap
+  x[i].market_value
+  x[i].complete = True
+---
+id: how many vehicles
+generic object: ALVehicleList
+question: |
+  How many vehicles do you have?
+fields:
+  - no label: x.target_number
+    datatype: integer
+---
+comment: |
+  Notes:
+  
+  * Do not try to use the `total()` method with an ALVehicle. ALVehicleList.
+  * The `.source` attribute is automatically 'vehicle'
+id: vehicle in list info
+generic object: ALVehicleList
+question: |
+  Tell us about your ${ ordinal( i ) } vehicle
+subquestion: |
+  % if len(x.complete_elements()) > 0:
+  So far you have told us about:
+  
+  % for item in x.complete_elements():
+  * ${ item.year_make_model() }
+  % endfor
+  % endif
+  
+fields:
+  - Year: x[i].year
+    input type: combobox
+    code: |
+      recent_years()
+  - Make: x[i].make
+    hint: e.g, Ford, Honda
+  - Model: x[i].model
+    hint: e.g., Fusion, Civic
+  - Current value: x[i].value
+    datatype: currency
+  - Loan balance: x[i].balance
+    datatype: currency
+  - Who owns this?: x[i].owner
+---
+# UNIQUE FOR ALSimpleValue
+---
+comment: |
+  Note: An ALSimpleValue doesn't require a transaction type in order to work.
+id: simple value
+generic object: ALSimpleValue
+question: |
+  Tell us about the value
+subquestion: |
+fields:
+  - Value: x.value
+    datatype: currency
+  - Transaction type: x.transaction_type
+    datatype: radio
+    required: False
+    choices:
+      - Expense: expense
+      - Income: income
+---
+# SHARED BY ALSimpleValueList
+---
+id: how many simple values
+generic object: ALSimpleValueList
+question: |
+  How many values do you have in your ${ x.object_name() } ${ x.__class__.__name__ }?
+fields:
+  - no label: x.target_number
+    datatype: integer
+    default: 2
+---
+# UNIQUE TO ALSimpleValueList
+---
+generic object: ALSimpleValueList
+code: |
+  x[i].value
+  x[i].complete = True
+---
+comment: |
+  Note: An ALSimpleValue doesn't require a transaction type in order to work.
+id: value list item value
+generic object: ALSimpleValueList
+question: |
+  Tell us about the ${ ordinal(i) } value for your ${ x.object_name() } ${ x.__class__.__name__ }
+subquestion: |
+fields:
+  - Source: x[i].source
+  - Value: x[i].value
+    datatype: currency
+  - Transaction type: x[i].transaction_type
+    datatype: radio
+    required: False
+    choices:
+      - Expense: expense
+      - Income: income
+---
+# TRANSLATIONS
+---
+depends on:
+  - user_language
+language: en
+variable name: job_items_names
+data:
+  full time: Full time wages
+  part time: Part time wages
+  tips: Tips
+  insurance: Insurance
+  federal_taxes: Federal taxes
+  state_taxes: State taxes
+  social_security: Social security
+  medicare: Medicare
+---
+depends on:
+  - user_language
+language: es
+variable name: job_items_names
+data:
+  full time: salario del el trabajo de tiempo completo
+  part time: salario del el trabajo de medio tiempo
+  tips: Propinas
+  insurance: Seguro
+  federal_taxes: Impuestos federales
```

```suggestion
---
variable name: job_items_names
data:
  full time: Full time wages
  part time: Part time wages
  tips: Tips
  insurance: Insurance
  federal_taxes: Federal taxes
  state_taxes: State taxes
  social_security: Social security
  medicare: Medicare
```

I don't think I realized when I first worked on this that the `data` block gets put in the translation file. This will make it more flexible and prevent errors if the `user_language` is set to `pt` or another language exclusive of `en` and `es`